rbac/server/api/proposals.py

```diff
@@ -192,5 +191,13 @@
     else:
         user_resource = await fetch_user_resource(conn, proposal_resource.get("object"))
         proposal_resource["approvers"] = [user_resource.get("manager")]
+    i = 0
+    approvers_count = len(proposal_resource["approvers"])
+    while i < approvers_count:
+        user_resource = await users_query.fetch_manager_chain(
+            conn, proposal_resource["approvers"][i]
+        )
+        proposal_resource["approvers"] = user_resource + proposal_resource["approvers"]
+        i += 1
     conn.close()
```

Comment:
Given that you're iterating over the current list of approvers and then adding each approver's manager chain, shouldn't this be de-duplicated instead of just adding the chain to the list? (5 levels up is a long way for most organizations, even if there were only two approvers explicitly assigned, 5 levels up their chain will likely result in duplicates.)